code/modules/mob/living/silicon/pai/emote.dm

```diff
@@ -86,7 +111,7 @@
 				to_chat(src, "You are not security.")
 
 		if ("help")
-			to_chat(src, "salute, bow, clap, flap, aflap, twitch, twitch_s, nod, beep, ping, \nbuzz, law, halt")
+			to_chat(src, "salute, bow, clap, flap, aflap, twitch, twitch_s, nod, beep, ping, \nbuzz, law, halt, meow1, meow2, squeal, squeak, softsqueak")
```

Comment:
In help it's listed as "meow1", in the above code it's listed as "meow"

Reply:
thank u soundscopes